StyleCop.Analyzers/StyleCop.Analyzers.Test/Verifiers/DiagnosticVerifier.cs

```diff
@@ -300,7 +300,7 @@ private static string FormatDiagnostics(ImmutableArray<DiagnosticAnalyzer> analy
                     {
                         Assert.True(
                             location.IsInSource,
-                            string.Format("Test base does not currently handle diagnostics in metadata locations. Diagnostic in metadata:\r\n", diagnostics[i]));
+                            string.Format("Test base does not currently handle diagnostics in metadata locations. Diagnostic in metadata:{0}\r\n", diagnostics[i]));
```

Comment:
:thought_balloon: It looks like the diagnostic was intended to be placed after the newline.